src/Components/Common/LanguageSelectorLogin.tsx

```diff
@@ -17,12 +18,16 @@ export const LanguageSelectorLogin = () => {
     }
   };
 
+  const availableLocales = Object.keys(LANGUAGE_NAMES).filter(
+    (l) => l === "en" || careConfig.availableLocales?.includes(l),
+  );
```

Comment:
why do we need a special case for en?

Reply:
So that even when there are no available languages set in env english is still there

Reply:
I think its safe to assume that the env will configured correctly, we also have a default set, so I think we can ignore it